modules/Auth/tests/auth_tests.cpp

```diff
@@ -356,6 +392,8 @@ TEST_F(AuthTest, test_faulted_state) {
     // wait until state faulted events arrive at auth module
     std::this_thread::sleep_for(std::chrono::milliseconds(100));
 
+    EXPECT_CALL(mock_publish_token_validation_status_callback, Call(_)).Times(0);
```

Comment:
since we're using a strick mock the test will fail on unexpected calls - we don't need here the check with times(0)

Reply:
Yes, indeed. I didn't use strict mock at the beginning